rafay/utils.go

```diff
@@ -2280,7 +2280,7 @@ func flattenWorkflowHandlerInline(input *eaaspb.WorkflowHandlerInline) []interfa
 		obj["config"] = flattenWorkflowHandlerConfig(input.Config, []interface{}{})
 	}
 	if len(input.Inputs) > 0 {
-		obj["inputs"] = flattenConfigContextCompoundRefs(input.Inputs)
+		obj["contexts"] = flattenConfigContextCompoundRefs(input.Inputs)
```

Comment:
`WorkflowHandler` is nothing but `Driver`, the config contexts attached to these are called `inputs` and not `contexts`.

Reply:
good catch, thanks for pointing it out.